livekit-agents/livekit/agents/pipeline/pipeline_agent.py

```diff
@@ -883,24 +883,25 @@ async def _execute_function_calls() -> None:
             if interrupted:
                 collected_text += "..."
 
-            msg = ChatMessage.create(text=collected_text, role="assistant")
-            self._chat_ctx.messages.append(msg)
+            if collected_text:
```

Comment:
this would skip `speech_handle.mark_speech_committed()`, do we want to mark it as committed before skipping the other operations?

Reply:
model's response contains empty content when it responds with a tool call, should we still mark it as committed? because after executing function model will respond with string that will be marked as committed